src/App.js

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-function App() {
+const App = () => {
+  const [newTodo, setNewTodo] = React.useState(''); 
   return (
     <div>
       <h1>Todo List</h1>
+      <AddTodoForm onAddTodo = {setNewTodo} />
       <AddTodoForm />
```

Comment:
Let's remove the AddTodoForm component that doesn't have the onAddTodo prop!

Reply:
Okie dokie! Just made changes.